core/components/atoms/input/Input.tsx

```diff
@@ -227,6 +228,7 @@ export const Input = React.forwardRef<HTMLInputElement, InputProps>((props, forw
     [styles['Input-icon--left']]: true,
     [styles['Input-icon--inputBlank']]: isInputBlank,
     [styles['Input-icon--error']]: error,
+    [styles['Input-iconLeft--tiny']]: size && size === 'tiny',
```

Comment:
Please directly use the `mr-3` classname in this case where we can't clearly follow the BEM convention